tinkerbell/stack/values.yaml

```diff
@@ -32,6 +32,7 @@ stack:
     roleBindingName: kube-vip-rolebinding
     # Customize the interface KubeVIP advertises on. When unset, KubeVIP will autodetect the interface.
     # interface: enp0s8
+    additionalEnv: []
```

Comment:
mind adding the code comment for the format of the values in this array? see the Smee example: https://github.com/tinkerbell/charts/blob/43a6c2ce1056f1d409ddef8ee7d45c33138e763c/tinkerbell/smee/values.yaml#L124